cmd/rofl/build/container.go

```diff
@@ -0,0 +1,177 @@
+package build
+
+import (
+	"context"
+	"encoding/base64"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+
+	coreCommon "github.com/oasisprotocol/oasis-core/go/common"
+	"github.com/oasisprotocol/oasis-core/go/common/cbor"
+	"github.com/oasisprotocol/oasis-core/go/common/version"
+	"github.com/oasisprotocol/oasis-core/go/runtime/bundle"
+	"github.com/oasisprotocol/oasis-sdk/client-sdk/go/connection"
+
+	buildRofl "github.com/oasisprotocol/cli/build/rofl"
+	"github.com/oasisprotocol/cli/cmd/common"
+	roflCommon "github.com/oasisprotocol/cli/cmd/rofl/common"
+	cliConfig "github.com/oasisprotocol/cli/config"
+)
+
+const (
+	artifactContainerRuntime = "rofl-container runtime"
+	artifactContainerCompose = "docker-compose.yaml"
```

Comment:
Last time I tried it (maybe ~year ago), `podman-compose` (and even `podman` in general) wasn't really a drop-in replacement to `docker-compose` in practice, and it was somewhat tedious migrating things to `podman`. There were a lot of minor incompatibilities.

I hope this has improved, otherwise it might be better to be very clear that `podman` is used to run the containers. Maybe even go as far to use [compose.yaml](https://github.com/compose-spec/compose-spec/blob/main/spec.md#compose-file) filename, which seems to be preferred in the latest spec and avoid mentioning docker.

Reply:
Good point, will rename it to `compose.yaml`.